entities/weapons/weapon_cs_base2/sv_commands.lua

```diff
@@ -1,5 +1,8 @@
 local meta = FindMetaTable("Player")
 function meta:dropDRPWeapon(weapon)
+    if !meta:IsValid() or !meta:IsPlayer() then return end
```

Comment:
This use of `meta` doesn't seem to be what you intend it to be. Do you mean to use `self` instead of `meta`?